students/barecki_andrzej/lesson_01_basics/lesson_01_basics_all_cases.py

```diff
@@ -0,0 +1,118 @@
+# Lesson 1 (Input, print and numbers)
+
+print('Problem 1: Sum of three numbers')
+print('Set first number:')
+a = int(input())
+print('Set second number:')
+b = int(input())
+print('Set third number:')
+c = int(input())
+print(a + b + c)
+
+print('\n\rProblem 2: Area of right-angled triangle')
```

Comment:
Rozumiem, że chcesz dodać nową linię. To zadziała, ale nie hardkodowałbym \n\r. Przychodzą mi na myśl 2 rozwiązania:
1. Rozbić to na 2 wywołania printa (ja bym tak zrobił):
`print()
print('\n\rProblem 2: Area of right-angled triangle')`
2. Użyć `os.linesep` (trzeba zaimportować os)

Reply:
usunalem \n\r i dodalem + os.linesep na koncu print(...... + os.linesep)
Poprawione.